package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "mock-jwks",
   "description": "A tool to mock a JWKS for development of microservices who consume jwts signed with RSA",
-  "version": "0.1.8",
+  "version": "0.1.9",
```

Please do not change the version. I do this when releasing.

```suggestion
  "version": "0.1.8",
```